### ISSUE TITLE
Building error

### ISSUE THREAD

**Issue by Tankerxyz** [huruji/eruda-webpack-plugin#2]
![image](https://user-images.githubusercontent.com/16082110/52700094-ee7c8200-2f7f-11e9-85ac-cdcc27d36758.png)
I just added 
```js
new ErudaWebpackPlugin({
      force: true
    })
```
to my webpack plugins array

**Comment by huruji** [comment]
can you tell me the version of your webpack @Tankerxyz

**Comment by huruji** [comment]
I am sorry, I've fixed that in [1.1.1](https://www.npmjs.com/package/eruda-webpack-plugin), it caused by an options which is missing `tool` field，like
```js
new erudaPlugin({
       force: true,
       plugins: []
}),
```
I've fixed that,see [ce37d0503df51f](https://github.com/huruji/eruda-webpack-plugin/commit/ce37d0503df51f4105a5a3ed9a1653154e7c62be)

Just try it again, thx @Tankerxyz
